test/users-query.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai';
+import { PrismaClient, User } from '@prisma/client';
+import { createAuthenticatedSession, createUsersQuery, executeGraphQLQuery } from './test-service.js';
+import { seedUsers } from '../scripts/seed-service.js';
+
+const prisma = new PrismaClient();
+
+export const usersQueryTests = (url: string) => {
+  describe('users query tests', () => {
+    beforeEach(async () => {
+      await prisma.user.deleteMany();
+    });
+
+    it('should return users ordered alphabetically with a limit', async () => {
+      const { token } = await createAuthenticatedSession('X Jonas Borges', 'jonas@teste.com', '2000-01-01', 'Test123');
+      const users = await seedUsers(10);
+      const usersQuery = createUsersQuery(5);
+
+      const response = await executeGraphQLQuery(url, usersQuery, token);
+
+      const usersData = response.data.data.users;
+      expect(usersData).to.have.lengthOf(5);
+      const expectedUsers = users.sort((a, b) => a.name.localeCompare(b.name)).slice(0, 5);
+      const usersDataNames = usersData.map((user: User) => user.name);
+      const expectedUsersNames = expectedUsers.map((user) => user.name);
+      expect(usersDataNames).to.deep.equal(expectedUsersNames);
```

Comment:
```suggestion
      const expectedUsers = users.sort(..).slice(0,5).map((it) => ({ id: it.id, name: it.name, ... }));
      expect(usersData).to.deep.equal(expectedUsers);
```
acho que você está se apegando demais ao nome haha
a ideia é buscar comparar todos os campos sempre que possível